docs/java/how-to-develop-a-workflow-definition-in-java.md

```diff
@@ -289,9 +289,9 @@ Related references:
 
 ### Calling other Workflows
 
-Workflows can invoke, and send Signals to, other Workflows purely by name. This helps particularly for executing Workflows written in other language SDKs.
+A Workflow can interact with other running Workflows (using Signals and Queries) or create instances of Child Workflows using `ExternalWorkflowStub`.
```

Comment:
I think this sentence is incorrect, for interacting with other running workflows yes use ExternalWorkflowStub, but for child workflows its ChildWorkflowStub.